teuthology/provision/cloud/openstack.py

```diff
@@ -21,26 +22,44 @@
 log = logging.getLogger(__name__)
 
 
+RETRY_EXCEPTIONS = (RateLimitReachedError, )
+
+
+def retry(function, *args, **kwargs):
+    """
+    Call a function (returning its results), retrying if any of the exceptions
+    in RETRY_EXCEPTIONS are raised
+
+    Note: does perform any additional logging when retrying
```

Comment:
presumably this means "does not", and it might not be bad to at least debug-log how many retries it took so we can see whether it's happening